### PR TITLE
[13.0][MIG] stock_inventory_chatter

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Standard stock_inventory_chatter migration to v13.

cc ~ @ForgeFlow @LoisRForgeFlow

### CODE REVIEW

File: stock_inventory_chatter/__manifest__.py
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/__manifest__.py]: keep the initial year: 2017-19

File: stock_inventory_chatter/__manifest__.py
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/__manifest__.py]: ForgeFlow

**Review by LoisRForgeFlow — COMMENTED** [review]
Remember to separate the pre-commit first run and the actual migration changes in 2 commits (https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch), this eases migration review.

File: stock_inventory_chatter/models/stock.py
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/models/stock.py]: actually this field is not present in any view, so it does not make much sense tu track it.

File: stock_inventory_chatter/models/stock.py
Comment (LoisRForgeFlow) [inline stock_inventory_chatter/models/stock.py]: Changing the locations doesn't produce any message in the chatter. Either implement it other way or remove this as it has no effect.

**Review by LoisRForgeFlow — COMMENTED** [review]
Some more comments after testing on runbot.

File: stock_inventory_chatter/data/stock_data.xml
Comment (HviorForgeFlow) [inline stock_inventory_chatter/data/stock_data.xml]: r/Canceled/Cancelled

File: stock_inventory_chatter/models/stock.py
Comment (HviorForgeFlow) [inline stock_inventory_chatter/models/stock.py]: Looking into Odoo v13 I noticed, you can never reach to a cancel state of a Inventory Adjustment with standard core modules.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by rousseldenis** [comment]
@LoisRForgeFlow Could you also check the box in https://github.com/OCA/stock-logistics-warehouse/issues/727 after merging? It is not (yet) automatic. Thanks

**Comment by LoisRForgeFlow** [comment]
@rousseldenis Sure, will do next time. Thanks

**Pull request merged**
